### PR TITLE
Added pyside2 support to backend.py and edited comments in qtDisplay.…

### PR DESCRIPTION
**Pull request by jrj99**
…py so that they reference Pyside2 as a supported GUI option. Also, I edited DataExchange.py so that it loads IGES files (as individual shapes) more efficiently. The old version of data exchange assumes that NB roots are never the same as NB shapes: I have found that when the two values are equal, NB roots are usually NB Shapes (check line 142 of DataExchange.py to see the changes)

### CODE REVIEW

**Comment by tpaviot** [comment]
thanks for all, I will check that in a couple of days

File: src/Extend/DataExchange.py [hunk 1]
```diff
@@ -61,319 +67,55 @@ def read_step_file(filename, return_as_shapes=False, verbosity=True):
...
     step_writer.Transfer(a_shape, STEPControl_AsIs)
     status = step_writer.Write(filename)
 
-    if not status == IFSelect_RetDone:
-        raise AssertionError("Error while writing shape to STEP file.")
-    if not os.path.isfile(filename):
-        raise AssertionError("File %s was not saved to filesystem." % filename)
-
-
-def read_step_file_with_names_colors(filename):
```
Comment (jf---) [inline src/Extend/DataExchange.py]: not sure why this method is deleted; it has no overlap with the `pyside2` backend

**Comment by jrj99** [comment]
I must have deleted those error messages by accident. apologies.

-Jared

**Comment by jrj99** [comment]
Any update on merging this?

**Comment by tpaviot** [comment]
two many unrequested changes in thie PR. I extracted PySide2 related code, and committed 0418babca48b024534003b74bfe69b56187e0581 thank you

**Pull request closed without merging**